doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -1077,6 +1076,16 @@ DFU libraries
 
 * Added a new DFU SMP target for the image update to an external MCU by using the MCUmgr SMP Client.
 
+Peripheral CPU DFU (PCD) library
+--------------------------------
+
+* Added a new function for reading peripherals CPU version number.
```

Comment:
```suggestion
* :ref:`subsys_pcd` library:

  * Added a new function for reading the CPU version number of a peripheral.
```
Do you want to mention the exact function?

Reply:
Sure we could refer to `int pcd_find_fw_version(void);`

Reply:
So, tag it like this: :c:func:`int pcd_find_fw_version(void);`